README.md

```diff
@@ -105,6 +105,14 @@ Boolean | `false` | >=3.4, 4.x, 5.x
 Configures rspec-puppet to use the `$trusted` hash when compiling the
 catalogues.
 
+#### trusted\_server\_facts
+Type    | Default | Puppet Version(s)
+------- | ------- | -----------------
+Boolean | `false` | >=4.3, 5.x
+
+Configures rspec-puppet to use the `$server_facts` hash when compiling the
```

Comment:
Where do you set the `$server_facts` hash? is it still mixed into the regular facts hash?

Reply:
@jeffberkley You don't... `$server_facts` is a static hash defined by the server construct.

See https://puppet.com/docs/puppet/5.3/lang_facts_and_builtin_vars.html#serverfacts-variable for example contents...

Reply:
@op-ct I'd like to say it was a concious decision to not allow over-riding the `$server_facts` values, as they can effectively be set by defining appropriate values in `let (:facts) { }`.

@rodjek / @DavidS - Thoughts?

Reply:
@fatmcgav Merit1: Configuring puppet with puppet https://github.com/simp/pupmod-simp-pupmod/blob/master/spec/classes/master_spec.rb#L19

Merit 2: Things that use information from `server_facts`: https://github.com/simp/pupmod-simp-krb5/pull/39/files#diff-a446e4110d88d59f153d0721dad568feR27

This is not insurmountable (as demonstrated, but SUPER irritating).